utils/roadie-backstage-entity-validator/src/validator.js

```diff
@@ -60,7 +60,7 @@ function modifyPlaceholders(obj) {
         }
       } catch (e) {
         throw new Error(
-          `Placeholder with name '${k}' is empty. Please remove it or populate it.`,
+          `Placeholder with name '${k}' is empty. Please remove it or populate it. ${e}`,
```

Comment:
Why did we decide to add this? Would this end up in the GitHub message of the GitHub validator action?